server/sequelize/migrations/20230804001548-create-tokens-table.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("Tokens", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      token: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
```

Comment:
A User can have many tokens as when the token expires Google will generate a new one